spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -92,6 +92,11 @@ public static File screenshot(Activity activity, String tag, String testClassNam
 
   private static void takeScreenshot(File file, final Activity activity) throws IOException {
     View view = activity.getWindow().getDecorView();
+    if (view.getWidth() == 0 || view.getHeight() == 0) {
+      Log.e(TAG, "Your view has no height or width. Are you sure "
```

Comment:
seems like it'd be better to just crash here (with the clarifying message) instead of waiting for Bitmap.createBitmap to crash